ajax/libs/upng-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "upng-js",
+  "description": "Small, fast and advanced PNG / APNG encoder and decoder",
+  "version": "2.0.0",
+  "author": "photopea",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/photopea/UPNG.js"
```

Comment:
Let's use git+https://github.com/photopea/UPNG.js.git
As the same with upstream from pm.
Thanks